resources/views/success.blade.php

```diff
@@ -0,0 +1,21 @@
+@extends('rapidez::layouts.app')
+
+@section('title', 'Checkout')
+
+@section('content')
+    <div class="container" :set='order=@json($t_order)' :set2='order.customer_email=window.localStorage.email'>
+        <graphql
+            :query='`query { customer { orders ( filter: { number: { eq:"${order.increment_id}" } } ) { items { status } } }}`'
+            v-cloak
+        >
+            <div slot-scope="{ data, runQuery, completed }" :set="completed=data?.customer?.orders?.items?.length&&['Voltooid', 'Completed'].includes(data.customer.orders.items[0].status)">
+                <div v-if="completed">
+                    @include('rapidez::checkout.steps.success')
+                </div>
+                <div v-else :set="window.setTimeout(()=>(completed||!data)?null:runQuery(), 2000)">
```

Comment:
This little bit here should probably become a vue component :)

But before I do that, first and foremost there needs to be a solution to get the order status for guests. Depending on how that's achieved, this can be properly reworked accordingly after that.